packages/js/client-config-builder/package.json

```diff
@@ -16,7 +16,8 @@
     "lint": "eslint --color -c ../../../.eslintrc.js src/",
     "test": "jest --passWithNoTests --runInBand --verbose",
     "test:ci": "jest --passWithNoTests --runInBand --verbose",
-    "test:watch": "jest --watch --passWithNoTests --verbose"
+    "test:watch": "jest --watch --passWithNoTests --verbose",
+    "build:readme": "yarn doc-snippets combine"
```

Comment:
I worry that this "build:readme" can be easily forgotten about.

I think for inspiration we can  look at how we build the CLI package. We have a `build` command which does a lot of different build steps, and then there is `build:fast` which only builds the source files.

Maybe a similar approach here would make sense?

Reply:
done!